### ISSUE TITLE
Warning about import_module deprecation

### ISSUE THREAD

**Issue by FireZenk** [kraiz/django-crontab#25]
````
/myapp/lib/python2.7/site-packages/django_crontab/crontab.py:13: 
RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils.importlib import import_module
````

Just to warn

**Comment by kraiz** [comment]
Thanks for bumping. Prepared a new version that drops support for old django versions but add 1.8 and 1.9. I'll wait for django 1.9 to get out before releasing.

**Comment by kraiz** [comment]
Released to pypi as 0.7.0
